controllers/semeru_compiler.go

```diff
@@ -79,6 +79,28 @@ func (r *ReconcileWebSphereLiberty) reconcileSemeruDeployment(wlva *wlv1.WebSphe
 	limitsMemory := getQuantityFromLimitsOrDefault(instanceResources, corev1.ResourceMemory, "1200Mi")
 	limitsCPU := getQuantityFromLimitsOrDefault(instanceResources, corev1.ResourceCPU, "8000m")
 
+	// Liveness probe
+	livenessProbe := corev1.Probe{
+		ProbeHandler: corev1.ProbeHandler{
+			Exec: &corev1.ExecAction{
+				Command: []string{"/bin/bash", "-c", "tail -10 /tmp/output.log*"},
+			},
+		},
+		InitialDelaySeconds: 10,
+		PeriodSeconds:       10,
+	}
+
+	// Readiness probe
+	readinessProbe := corev1.Probe{
+		ProbeHandler: corev1.ProbeHandler{
+			Exec: &corev1.ExecAction{
+				Command: []string{"/bin/bash", "-c", "grep -q '#INFO:  StartTime' /tmp/output.log*"},
+			},
+		},
+		InitialDelaySeconds: 10,
```

Comment:
The initial delay should be shortened, so that the container becomes ready sooner. Suggest initial delay of 5 seconds, period seconds of 5 seconds and set the failure threshold to twich the default value.